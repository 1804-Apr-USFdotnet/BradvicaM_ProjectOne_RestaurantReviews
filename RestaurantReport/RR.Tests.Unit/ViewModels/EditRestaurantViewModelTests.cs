using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.ViewModels;

namespace RR.Tests.Unit.ViewModels
{
    [TestClass]
    public class EditRestaurantViewModelTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void ToString_OnCall_ReturnsCorrectInformation()
        {
            var viewModel = new EditRestaurantViewModel
            {
                Name = "Some Name",
                City = "Some City",
                PhoneNumber = "1234567890",
                RestaurantPublicId = Guid.Empty,
                State = "CA",
                Street = "123 Whatever Dt.",
                Website = "www.blah.com",
                ZipCode = 12345
            };

            Approvals.Verify(viewModel);
        }
    }
}
