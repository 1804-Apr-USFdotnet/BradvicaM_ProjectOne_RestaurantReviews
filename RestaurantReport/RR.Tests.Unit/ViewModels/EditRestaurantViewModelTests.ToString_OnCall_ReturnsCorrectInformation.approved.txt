
RestaurantPublicId: 00000000-0000-0000-0000-000000000000
Name: Some Name
Street: 123 Whatever Dt.
City: Some City
State: CA
ZipCode: 12345
PhoneNumber: 1234567890
Website: www.blah.com
